data/themes/default/style.css

```diff
@@ -540,6 +540,41 @@ BBTCOView {
 	qproperty-textColor: rgb( 255, 255, 255 );
 }
 
+/* subwindows in MDI-Area */
+SubWindow {
+        color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
+                                                stop: 0 #4b525c, stop: 1.0 #31363d);
+        qproperty-activeColor: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
+                                                stop: 0 #33383e, stop: 1.0 #1a1c20);
+        qproperty-textShadowColor: rgb(0, 0, 0);
+        qproperty-borderColor: rgb(0, 0, 0);
+}
+
+/*SubWindow Title Text */
+SubWindow > QLabel {
+        color: rgb(255, 255, 255);
+        font-size: 12px;
+        font-style: normal;
+        font-weight: bold;
```

Comment:
I just noticed that the text before wasn't bold. I'd get it back to regular weight, unless you think it increases readability.